sdk/storage/azure-storage-file-datalake/tests/test_file_async.py

```diff
@@ -309,21 +362,29 @@ async def test_append_data_async(self, datalake_storage_account_name, datalake_s
         # Act
         response = await file_client.append_data(b'abc', 0, 3)
 
-        self.assertIsNotNone(response)
+        assert response is not None
 
     @DataLakePreparer()
-    async def test_append_empty_data_async(self, datalake_storage_account_name, datalake_storage_account_key):
+    @recorded_by_proxy_async
+    async def test_append_empty_data(self, **kwargs):
+        datalake_storage_account_name = kwargs.pop("datalake_storage_account_name")
+        datalake_storage_account_key = kwargs.pop("datalake_storage_account_key")
+
         await self._setUp(datalake_storage_account_name, datalake_storage_account_key)
         file_client = await self._create_file_and_return_client()
 
         # Act
         await file_client.flush_data(0)
         file_props = await file_client.get_file_properties()
 
-        self.assertIsNotNone(file_props['size'], 0)
+        assert file_props['size'] is not None
```

Comment:
Seems this assert was supposed to be equality check with `0`. Would you mind updating this. Likely applies to sync as well.